ngraph/test/layout.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/core/layout.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace ov;
+
+TEST(layout, basic) {
+    Layout l = "NcDHw";
```

Comment:
Why do you use small `c`?

Reply:
This also verifies that names are case insensitive, e.g. both 'C' and 'c' represent channels